logstash-plugins/logstash-integration-logstash
# Apply 9800 default port, introduce `hosts` config for `logstash-output`.

## Release notes

[rn:skip]

## What does this PR do?
This PR covers two main changes:
- applies a default 9800 port on both `logstash-input` (aka server or downstream input) and `logstash-output` (aka client or upstream output)
- removes (since we are early stage of plugin development) `host`, introduces `hosts` (<scheme:host:port> prototyped) option in `logstash-output`

## Why is it important/What is the impact to the user?
- We are going to support HA in the nearest future and `host` will be deprecated, instead using `hosts` from beginning will be good start
- From user perspective, minimal change will be better, so default port and URI normalization (apply scheme based on `ssl_enabled`, use default port) is important

## Checklist


## Author's Checklist

## How to test this PR locally
- Unit tests will ensure that applied changes are intentional and enforce the expectation
- Clone the change locally and run `logstash-input` and `logstash-output` together

## Related issues
- Closes #12 

## Use cases

## Screenshots

## Logs
